source/_reference-a-f.html.md.erb

```diff
@@ -3106,7 +3131,7 @@ A column is a component of a sheet or report.
 **tags** | string[] | Set of tags to indicate special columns. Each element in the array is set to one of the following values:<ul><li>**CALENDAR_END_DATE**</li><li>**CALENDAR_START_DATE**</li><li>**CARD_DONE**</li><li>**GANTT_ALLOCATION**</li><li>**GANTT_ASSIGNED_RESOURCE**</li><li>**GANTT_DISPLAY_LABEL**</li><li>**GANTT_DURATION**</li><li>**GANTT_END_DATE**</li><li>**GANTT_PERCENT_COMPLETE**</li><li>**GANTT_PREDECESSOR**</li><li>**GANTT_START_DATE**</li></ul>
 **title** | string | Column title
 **validation** | Boolean | Indicates whether validation has been enabled for the column (value = **true**)
-**version** | number | A zero-based integer indicating whether the column allows multi-contact data. A value of **0** indicates the data will be stored as text and the column type will be TEXT_NUMBER, a value of **1** for sheets and reports or a value of **2** for Sights (aka dashboards) indicates the data will be stored as contacts and the column type will be MULTI_CONTACT_LIST. If not specified, the default value is 0 for sheets and reports or 1 for Sights (aka dashboards). 
+**version** | number | Read only. Indicates whether a column supports new column types. Each element in the array is set to one of the following values:<ul><li>**0**: TEXT_NUMBER, CONTACT_LIST, or PICKLIST</li><li>**1**: MULTI_CONTACT_LIST</li><li>**2**: MULTI_PICKLIST</li></ul>See [Versioning and Changes](#versioning-and-changes).
```

Comment:
version tells you which "column type level" aka "version" your column is. If you have a Contact List, you have a version 0 column, if you have a MultiContactList, you have a version 1 column, and if you have a multipicklist, you have a version 2 column.

Reply:
I feel like "indicates whether a column supports new column types" feels a little awkward.